penpost/penpost-web
# Gaj/feature/circle ci config

#### Purpose:
 > Check all that apply.

### What does this PR do?
 > Clearly and concisely describe changes made. 

Applies commands for cypress testing. 

Issues closed #9 

### Problems Encountered & Resulting Solutions  
> State any problems encountered 

NA

> Describe your code changes in detail for reviewers.

NA

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

see `.circleci/config.yml`

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Please see pipeline in circleci.

### Steps To Take
> Any questions and possible steps moving forward.

Should other commands be applied?
